app/views/statuses/edit.html.erb

```diff
@@ -27,19 +27,20 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
-<% html_title t(:label_administration), "#{t(:label_edit)} #{Status.model_name.human} #{h @status.name}" %>
+<% html_title t(:label_administration), "#{t(:label_edit)} #{Status.model_name.human} #{h @status.name_was}" %>
```

Comment:
It existed before but the `h` can by now be removed since the string will be escaped anyway.

Reply:
I tried but this does not work as expected:

with `h`:

![image](https://github.com/user-attachments/assets/20715c06-f491-4afe-bb94-595084bd0af6)

without `h`:

![image](https://github.com/user-attachments/assets/9773e150-3fa4-4c0f-8479-18abd26da63a)

Note the difference in the tab title in the left sidebar.

So it's still valid, there no XSS injection, but for consistency, if the tags are escaped everywhere (header, breadcrumbs, etc), they should be escaped in the title as well, not stripped.

About where it's done: the string is used to set the title using meta_tags, and this meta_tags calls `strip_tags` at some point (when calling [`#normalize_title`](https://github.com/kpumuk/meta-tags/blob/00358f4160eafe3e5c21cb9466a5d794e541b5e2/lib/meta_tags/text_normalizer.rb#L8-L34)).